### ISSUE TITLE
Bug: Stationsname nach dem Checkout ist falsch.

### ISSUE THREAD

**Issue by f2k1de** [derf/travelynx#157]
Ich fahre mit einer U5 zum Berliner Hbf im BVG Backend.
Nach dem Checkout wird mir als Station Elsterwerda, Bahnhof angezeigt statt Berlin Hauptbahnhof. Die angezeigten weiteren Verbindungen sind aber die richtigen von Berlin Hauptbahnhof. 
![IMG_2927](https://github.com/user-attachments/assets/27719dcb-0b04-4b89-8384-fa9941c4bb6f)

**Comment by homologic** [comment]
Das passiert auch in Kostrzyn, wenn man da hinfährt kommt sobwohl im IRIS als auch im DB-HAFAS Hostetin als Ziel raus.

**Comment by derf** [comment]
Jo, die Angabe des angefragten Stationsnamen im HAFAS ist ein bisschen kaputt. Das betrifft aber nur die Anzeige im Frotend, nicht die Daten im Backend.

**Comment by homologic** [comment]
Der Bug tritt übrigens auch im IRIS auf, siehe hier zum Beispiel:

![image](https://github.com/user-attachments/assets/bfcc601e-99c5-4e3a-8426-b94781d0ea81)

Ich werde mich morgen mal ransetzen und analysieren wo das herkommt, in der Stationsliste ist ja alles richtig.

**Comment by Ein-Tim** [comment]
Betrifft dieses Problem auch das bahn.de Backend?
